package.json

```diff
@@ -23,6 +23,7 @@
     "demo": "run-os",
     "demo:win32": "npm run prebuild && cross-env ELECTRON_DEV=true electron . --url=file:///src/demo/index.html",
     "demo:darwin": "npm run prebuild && cross-env ELECTRON_DEV=true electron . --url=file://$(pwd)/src/demo/index.html",
+    "format": "pretty-quick",
```

Comment:
`pretty-quick` is this a third party dependency? If yes, I don't see it listed in dev dependencies

Reply:
Yes, sorry, this was already pushed to master -> https://github.com/symphonyoss/SymphonyElectron/blob/91dba95cbd91cd53270292462b5f44a655012f56/package.json#L148